### PR TITLE
feat: TUP-704 improve Manage Account markup

### PR DESCRIPTION
**Pull request by wesleyboar**
## Overview

Improve markup and styles for Manage Account.

## Related

- improve markup since:
    - [TUP-704](https://tacc-main.atlassian.net/browse/TUP-704)
    - #431

## Testing

1. Visit http://localhost:8000/portal/account.
2. Review page layout and styles.
3. Review markup.

## Changes

- different markup
- slightly different spacing
- slightly larger headings — [designer approved](https://tacc-team.slack.com/archives/G01K9ET7JM7/p1709848753119239?thread_ts=1709846775.356789&cid=G01K9ET7JM7)

## UI

| Before | After |
| - | - |
| new with tiny headings | h2 border - small font |

### CODE REVIEW

File: libs/tup-components/src/accounts/ManageAccount.module.css [hunk 1]
```diff
@@ -31,36 +30,37 @@
...
-    padding-top: 10px;
     padding-bottom: 10px;
-    margin-bottom: 1em;
     border-bottom: 1px solid #707070;
 }
-.tap-description {
-    margin-block: 1em;
+.tap-header:is(h2) {
+    font-size: var(--global-font-size--small);
 }
```
Comment (wesleyboar) [inline libs/tup-components/src/accounts/ManageAccount.module.css:44]: **Why?** Cuz we should use `<h2>` for this **heading**, but Designer doesn't want the large default size of `<h2>`.

**Review by wesleyboar — COMMENTED** [review]
Notes for reviewers.

File: libs/tup-components/src/accounts/ManageAccount.module.css [hunk 2]
```diff
@@ -31,36 +30,37 @@
...
+.tap-action {
     /* To place actions inline and add space between them */
     display: inline-block;
     margin-right: 1em;
     margin-bottom: 1em;
 }
+.tap-action:is(button),
+.tap-action > button {
+    width: 200px;
+}
```
[the remark this answers is not in the document]
Comment (R-Tomas-Gonzalez) [reply libs/tup-components/src/accounts/ManageAccount.module.css:58]: Seen. 

I laughed at "nor sensible markup".

**Review by R-Tomas-Gonzalez — APPROVED** [review]
I think this all looks clean. Just looking at the first article's markup is pleasing to the eye. 

Approved

**Pull request merged**